templates/asyncapi1/main.dot

```diff
@@ -36,7 +36,7 @@ Base URLs:
 ## {{=t}}
 
 {{? data.topic.parameters }}
-<h3 id="{{=data.topicName}}-parameters">Parameters</h3>
+### Parameters
```

Comment:
What particular Javascript errors were you seeing here? This change will have the effect that all `Parameters` entries in the TOC will jump to the first `Parameters` heading (I think).

Reply:
When I am using paramters without the fix I am getting the following JavaScript errors:

![paramters-js-error](https://user-images.githubusercontent.com/1290363/39870788-df9e5df6-5462-11e8-8ce2-460aacf7043b.png)

This seems to appear because jQuery selectors do not allow curly braces without escaping them.
>To use any of the meta-characters ( such as  !"#$%&'()*+,./:;<=>?@[\]^`{|}~ ) as a literal part of a name, it must be escaped with with two backslashes: \\. For example, an element with id="foo.bar", can use the selector $("#foo\\.bar").

[jQuery Selectors](http://api.jquery.com/category/selectors/)

You are right the entries in the TOC will jump to the first Parameters heading. I missed that. The same issue seems to appear for schema properties and message names. So I would suggest to adjust the templates to something like that:
```
<h3 id="{{=data.topicName.replace(/\W/g,'-')}}-parameters">Parameters</h3>
```

I am not so happy with using HTML Tags in a markdown document, it just feels like a workaround but I am by far not familiar with markdown and cannot find another solution...